isis/src/qisis/apps/ipce/IpceMainWindow.cpp

```diff
@@ -225,8 +225,11 @@ namespace Isis {
    * @param view QWidget* The view to close.
    */
   void IpceMainWindow::removeView(QWidget *view) {
-    view->close();
+    QDockWidget *parentDock = qobject_cast<QDockWidget *>(view->parent());
```

Comment:
Is there always a parent dock widget when this is called? Is `view->close()` ever still necessary to call?

Reply:
Yes, there will always be a parent dock widget because a view is stored inside of a QDockWidget when it is opened. When we call removeDockWidget(), this closes the dock widget and its children, including the view.  So, we do not need to call view->close() as well.

Reply:
:+1: Thanks!